### ISSUE TITLE
read_files test coverage

### ISSUE THREAD

**Issue by DJuboor** [lbanawa/photo_filters#4]
**TL;DR:**
Write tests for the read_files function and update the function to handle these test cases when applicable.

**Objective:**
In src/processing.py, identify any _reasonable_ unhandled edge cases that may break the read_files function. Modify the read_files function to better handle these edge cases by returning a False status from the function and a helpful message to the user. Write unittests in tests/test_process.py to simulate an input that would cause the edge case. 

**Suggested Tests:**
Fail Cases:
- [x] input is not a string
- [x] input is not a valid path
- [x] input is not an image file

Success Case:
- [x] input is a valid path with an image in it, returns True and a list of an array / the file path

**Success criteria:**
- all ^ test cases are well handled. 
- all ^ tests written in tests/test_process.py and able to be successfully run via run_tests.py

**Comment by lbanawa** [comment]
1c16a755496df40e603d4d6fc7d107e5b0797b90

Test for input is not an image file

**Comment by lbanawa** [comment]
91a094bd3e73facaadd2ede185f6f46cd494b6e5

Test for success case

**Comment by lbanawa** [comment]
Ticket resolved: https://github.com/lbanawa/photo_filters/pull/7
